samples/ControlCatalog/App.xaml

```diff
@@ -22,6 +25,26 @@
     <Style Selector="Label.h3">
       <Setter Property="FontSize" Value="12" />
     </Style>
-    <StyleInclude Source="/SideBar.xaml"/>
+    <StyleInclude Source="/SideBar.xaml" />
   </Application.Styles>
+  <TrayIcon.Icons>
+    <TrayIcons>
+      <TrayIcon Icon="/Assets/test_icon.ico" ToolTipText="Avalonia Tray Icon ToolTip">
+        <NativeMenu.Menu>
```

Comment:
Why is it an attached property there? It's an integral part of the icon.

Reply:
made it a property.